airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -240,10 +241,10 @@ def _get_init_action_timeout(self):
         match = re.match(r"^(\d+)([sm])$", self.init_action_timeout)
         if match:
             if match.group(2) == "s":
-                return self.init_action_timeout
+                return {"seconds": int(self.init_action_timeout)}
```

Comment:
```suggestion
                return {"seconds": int(float(match.group(1))}
```

Reply:
Actually, converting to float first seems to be unnecessary, as a float value will not match a regex anyway.

Reply:
Not sure if I understand

Reply:
You cannot convert `init_action_timeout` to `int`, as `init_action_timeout` has a character `s` at the end. You have to convert `match.group(1)` instead.

Reply:
@turbaszek it seems you have reverted to the previous version here in one of latter commits :)

Reply:
Fixed